projects/plugins/jetpack/modules/subscriptions/subscribe-overlay/subscribe-overlay.css

```diff
@@ -24,13 +24,20 @@ body.jetpack-subscribe-overlay-open {
 	visibility: hidden;
 	overflow: hidden;
 	top: 100%;
-	margin: 15% auto;
+	/*margin: 15% auto;*/
 	width: 100%;
-	max-width: 400px;
+	/*max-width: 400px;*/
```

Comment:
Would be good to get this content width happen via blocks directly; I get it's important to have or otherwise long texts look weird on wide modal.